### PR TITLE
Fix not sending the correct xem url

### PR DESCRIPTION
**Pull request by duramato**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: medusa/scene_exceptions.py [hunk 1]
```diff
@@ -38,6 +38,7 @@
 exceptions_cache = defaultdict(lambda: defaultdict(set))
 exceptionLock = threading.Lock()
 
+VALID_XEM_ORIGINS = {'anidb', 'tvdb', }
```
Comment (medariox) [inline medusa/scene_exceptions.py]: Was this meant to be a tuple instead?

Comment (medariox) [reply medusa/scene_exceptions.py]: ^ @duramato

Comment (duramato) [reply medusa/scene_exceptions.py]: Not my doing, @labrys

**Comment by labrys** [comment]
@medariox nope a set

File: medusa/scene_exceptions.py (hunk 1 above)
Comment (labrys) [reply medusa/scene_exceptions.py]: Nope it was meant to be a set

**Pull request merged**